src/Router/VersionedRouter.php

```diff
@@ -71,8 +72,8 @@ function ($val) {
         }
 
         if ($badMethod) {
-            throw new Exception('Method not supported', 415);
```

Comment:
This should probably be `405` instead, shouldn't it?

Reply:
Agreed